src/context.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { usersInfo } from "./Utils/constants";
+
+export const UserContext = React.createContext();
+
+export const Provider = (props) => {
```

Comment:
change name of the provider to be more specific. As `Provider` is very abstract name